nimare/version.py

```diff
@@ -61,4 +61,9 @@
 VERSION = __version__
 REQUIRES = ["nibabel", "numpy", "scipy", "pandas", "statsmodels", "nipype",
             "scikit-learn", "nilearn", "duecredit", "pyneurovault", "six",
-            "matplotlib", "nltk", "fuzzywuzzy", "cognitiveatlas"],
+            "matplotlib", "nltk", "fuzzywuzzy", "cognitiveatlas", "tdqm"]
```

Comment:
I believe this should be `tqdm`, not `tdqm`.

Reply:
done